<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add</class>
 <widget class="QDialog" name="add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>438</width>
    <height>484</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>438</width>
    <height>484</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加联系人</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加联系人</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>编号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_id">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>请输入编号</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,7,1">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_name">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入姓名</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,7,1">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>类别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_leibie">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>家人</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>朋友</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>同学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>同事</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>其他</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_male">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>男</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_fmale">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>女</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,7,1">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>电话号码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_phonenum">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入电话号码</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,7,1">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>电子邮箱：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_mail">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入电子邮箱</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_ok">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_cancle">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
